cl/simple_pages/templates/v2_help/index.html

```diff
@@ -8,11 +8,11 @@
 
 
 {% block content %}
-  <div class="hidden-xs col-sm-2 col-md-3"></div>
+  <div class="text-sm-semibold text-primary-600">COURTLISTENER</div>
```

Comment:
We can use CSS to do the uppercasing here. It's better because:

1. It can be copy/pasted and the user will get CourtListener instead of COURTLISTENER (neat). 

2. Developers can't turn it off once it's componentized. All eyebrows will be uppercase as enforced by the component.